js/index.js

```diff
@@ -0,0 +1,118 @@
+      name: 'Mosh Hamedani',
+      info: 'Passionate Software Engineer and Best-selling Author',
+      detailedInfo:
+        "Hi! My name is Mosh (Moshfegh) Hamedani! I'm a passionate software engineer with two decades of experience and I've taught over 3 million people",
+    },
+    {
+      image: 'assets/images/instructor4-maximilianschwarzmüller.jpg',
+      name: 'Maximilian Schwarzmüller',
+      info: 'Professional Web Developer and Instructor',
+      detailedInfo:
+        'As a self-taught professional I really know the hard parts and the difficult topics when learning new or improving on already-known languages',
+    },
+    {
+      image: 'assets/images/instructor5-bentristem.jpg',
+      name: 'Ben Tristem',
+      info: 'GameDev.tv Founder :: Entrepreneur :: Passionate Teacher',
+      detailedInfo:
+        "Hi, I'm Ben. I have a degree in computing from Imperial College London, and a physics diploma from the Open University",
+    },
+    {
+      image: 'assets/images/instructor6-coltsteele.jpg',
+      name: 'Colt Steele',
+      info: 'Developer and Bootcamp Instructor',
+      detailedInfo:
+        "Hi! I'm Colt. I'm a developer with a serious love for teaching. I've spent the last few years teaching people to program at 2 different immersive bootcamps",
+    },
+  ];
+  function generateInstructors() {
+    instructors.forEach((ins, ind) => {
+      const li = document.createElement('li');
+      li.classList.add(
+        'speaker',
+        'd-flex',
+        'p-4',
+        'align-items-center',
+        'col-md-6',
+      );
+      if (ind >= 2) {
+        li.classList.add('d-none');
+      }
+      // Create Speaker Image div
+      const divSpeakerImage = document.createElement('div');
+      divSpeakerImage.classList.add(
+        'col-5',
+        'col-lg-auto',
+        'ps-2',
+        'speaker__image',
+      );
+
+      const speakerImage = document.createElement('img');
+      speakerImage.src = ins.image;
+      speakerImage.alt = `${ins.name} Icon`;
+
+      divSpeakerImage.appendChild(speakerImage);
+      // Create Info div
+      const infoDiv = document.createElement('div');
+      infoDiv.classList.add('p-2');
+
+      const speakerName = document.createElement('h3');
+      speakerName.classList.add('speaker__name');
+      speakerName.textContent = ins.name;
+      const speakerInfo = document.createElement('p');
+      speakerInfo.classList.add('red-color', 'speaker__info');
+      speakerInfo.textContent = ins.info;
+      const speakerDetailedInfo = document.createElement('p');
+      speakerDetailedInfo.classList.add('dark-gray', 'mt-3');
+      speakerDetailedInfo.textContent = ins.detailedInfo;
+
+      infoDiv.appendChild(speakerName);
+      infoDiv.appendChild(speakerInfo);
+      infoDiv.appendChild(speakerDetailedInfo);
+
+      li.appendChild(divSpeakerImage);
+      li.appendChild(infoDiv);
+
+      ul.appendChild(li);
+    });
+  }
+  generateInstructors();
```

Comment:
- [ ] Please make use of the `window.onload` function to make sure that it works after the page assets are successfully loaded.

Reply:
Done